src/Contao/SessionStorage.php

```diff
@@ -85,7 +118,8 @@ public function createInstance($key)
     public function has($name)
     {
         $this->load();
-        return isset($this->attributes[$name]);
+
+        return (bool) $this->attributes[$name];
```

Comment:
This will trigger a notice if not set, we should not introduce notices!

Reply:
I have rework this

Reply:
13772ea3